src/utils/email.js

```diff
@@ -0,0 +1,16 @@
+import sendGrid from '@sendgrid/mail';
+
+export default async (apiKey, payload) => {
+  sendGrid.setApiKey(apiKey);
+  const data = await sendGrid.send(payload);
+  return data;
+};
+
+
+// const msg = {
```

Comment:
@Simplemart17 Why are you commenting out this line, could it be better if it is removed?

Reply:
It has been resolved. Thanks